BitwardenShared/Core/Platform/Services/ConfigService.swift

```diff
@@ -187,6 +203,10 @@ class DefaultConfigService: ConfigService {
         return configuration?.featureStates[flag]?.stringValue ?? defaultValue
     }
 
+    func getRemoteFeatureFlags() async -> [FeatureFlag: AnyCodable] {
```

Comment:
🤔 Does it make sense to also gate this behind the `DEBUG_MENU` flag?

Reply:
yeah, good call out.  That would make the most sense to me to avoid any issues in release builds